common/local_filestore.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_filestore tool."""
+
+import os
+
+from common import new_process
+
+
+def local_filestore_command(arguments, expect_zero=True):
```

Comment:
"arguments" doesn't make sense here like it does for gsutil.py (too much copy pasting).
In fact, I think we can probably get rid of this function as it doesn't do anything in `local_filestore` (unlike in `gsutil`). Just replace it with `new_process.execute`

Reply:
Since it relates to a unit test in #422, let me get back to this later.

Reply:
In #422, I use this function to do a unit test that, `parallel` is not passed to execution even when outside explicitly set `parallel` as true. Also, accordingly, there is a unit test checking, in cloud filestore mode, the `parallel` does pass to this layer.

Reply:
This is a problem with doing concurrent PRs where one depends on another. It's fine to not change filestore_utils if this PR only deals with local_filestore.
But if we can't change `local_filestore` why even bother seperating this from #422?

Reply:
All right. Let me resolve this and I will change the code for #422 Later.

Reply:
Done.